File: src/de/carste_lenz/liquibasegui/ui/OpenCommandViewHandler.scala
package de.carste_lenz.liquibasegui.ui

import org.eclipse.core.commands._
import org.eclipse.ui.handlers._

class OpenCommandViewHandler extends AbstractHandler {

  def execute(event: ExecutionEvent): Object = {
      val page = HandlerUtil.getActiveWorkbenchWindow(event).getActivePage;
      page.showView(CommandView.ID)
      
      null
  }

}

File: de.carsten_lenz.liquibasegui/src/de/carsten_lenz/liquibasegui/ui/OpenCommandViewHandler.java
package de.carsten_lenz.liquibasegui.ui;

import org.eclipse.core.commands.AbstractHandler;
import org.eclipse.core.commands.ExecutionEvent;
import org.eclipse.core.commands.ExecutionException;
import org.eclipse.ui.IWorkbenchPage;
import org.eclipse.ui.PartInitException;
import org.eclipse.ui.handlers.HandlerUtil;

class OpenCommandViewHandler extends AbstractHandler {

    @Override
    public Object execute(ExecutionEvent event) throws ExecutionException {
        IWorkbenchPage page = HandlerUtil.getActiveWorkbenchWindow(event).getActivePage();
        try {
            page.showView(CommandView.ID);
        } catch (PartInitException e) {
        }

        return null;
    }

}